staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -280,6 +280,9 @@ type VirtualMachineInstanceStatus struct {
 	// Current topology may differ from the desired topology in the spec while CPU hotplug
 	// takes place.
 	CurrentCPUTopology *CPUTopology `json:"currentCPUTopology,omitempty"`
+
+	// EvictionStrategy shows the VM evictionStrategy
```

Comment:
is there a 
```
//+optional
```
required here? I see that its present for almost all fields that are optional except `CurrentCPUTopology`?

Reply:
The field always gets set so I'd say no? Do you think is required?